src/frameworks/index.ts

```diff
@@ -403,6 +411,7 @@ export async function prepareFrameworks(
       config.public = relative(projectRoot, hostingDist);
       if (wantsBackend) codegenFunctionsDirectory = codegenProdModeFunctionsDirectory;
     }
+    deployedFrameworks.push(`${framework}${codegenFunctionsDirectory ? "_ssr" : ""}`);
```

Comment:
What is `codegenFunctionsDirectory` typically, here? This _looks_ like it could potentially contain user data which makes me ask the question.

Reply:
It has the customer’s directory structure, but we’re not actually using it anywhere. We’re just testing whether it is undefined to decide whether to put the “_ssr” suffix on the framework